### DISCUSSION TITLE
Where there be more operators to the "documents where x equals y" effect? Also sorting?

### DISCUSSION THREAD

**Discussion by ReportCardsMC** [Romitou/MongoSK#13] [Q&A] [answered]
Title basically says most of it, I think it'd be nice having other operators like `where x greater than y` or `where x less than y` and so on.

And will there be an effect for mongo's sort function?

**Comment by Romitou** [comment, marked as answer]
> I think it'd be nice having other operators like where x greater than y or where x less than y and so on.

I agree with you, and as specified in https://github.com/Romitou/MongoSK/issues/7#issuecomment-745162805, I would indeed like to add soon all the comparators of MongoDB :

- `ALL`
- `EQUALS`
- `NOT_EQUAL`
- `EXISTS`
- `GREATER_THAN`
- `GREATER_THAN_OR_EQUAL`
- `IN`
- `LESS_THAN`
- `LESS_THAN_OR_EQUAL`

... I think they are the most important.
So yes, it's planned! 😄

> And will there be an effect for mongo's sort function?

This is something I hadn't thought of, and indeed it would be very useful. I'm going to add it to my to-do list! 🙂
